File: csharp/MapleStory.OpenAPI/Src/Dto/Union/UnionRaiderBlockControlPointDTO.cs
using Newtonsoft.Json;

namespace MapleStory.OpenAPI.Dto
{
    /// <summary>
    /// 유니온 블록 기준점 좌표 정보
    /// </summary>
    public class UnionRaiderBlockControlPointDTO
    {
        /// <summary>
        /// 블록 기준점 X좌표
        /// </summary>
        [JsonProperty("x")]
        public long X { get; set; }

        /// <summary>
        /// 블록 기준점 Y좌표
        /// </summary>
        [JsonProperty("y")]
        public long Y { get; set; }
    }
}

File: java/src/main/java/dev/spiralmoon/maplestory/api/tms/dto/union/UnionRaiderBlockControlPointDTO.java
package dev.spiralmoon.maplestory.api.tms.dto.union;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.ToString;

/**
 * 方塊基準點座標
 */
@AllArgsConstructor
@NoArgsConstructor
@Data
@ToString
public class UnionRaiderBlockControlPointDTO implements dev.spiralmoon.maplestory.api.common.dto.union.UnionRaiderBlockControlPointDTO {

    /**
     * 方塊基準點 x 座標
     */
    @JsonProperty("x")
    private long x;

    /**
     * 方塊基準點 y 座標
     */
    @JsonProperty("y")
    private long y;
}
